panther_analysis_tool/main.py

```diff
@@ -810,6 +828,61 @@ def setup_parser() -> argparse.ArgumentParser:
     return parser
 
 
+def zip_managed_schemas(args: argparse.Namespace) -> Tuple[int, str]:
+    """Packs managed schemas of a tagged release into a local zip file.
+
+    Args:
+        args: The populated Argparse namespace with parsed command-line arguments.
+
+    Returns:
+        A tuple of return code and the archive filename.
+    """
+
+    manifest = []
+    with tempfile.TemporaryDirectory(prefix="zip-managed-schemas-") as tmp_dir:
+        repo_url = "https://github.com/panther-labs/panther-analysis"
+        repo_dir = os.path.join(tmp_dir, "panther-analysis")
+
+        logging.info("Cloning %s tag of %s", args.release, repo_url, repo_dir)
+        result = subprocess.run([
+            "git", "clone", "--branch", args.release, "--depth", "1", "-c",
+            "advice.detachedHead=false", repo_url, repo_dir
+        ])
```

Comment:
checking that args.release is a sem ver might fix the ci error

Reply:
The error is confusing and of any value IMO.
I added a `check=True` to raise an exception on non-zero return code and a `timeout=120` to raise an exception if a process gets struck.
I resorted to `#nosec` since the only way to do this is by spawning `git`.